src/mock-server/mock/oppgaver.json

```diff
@@ -1,199 +1,2689 @@
 {
     "data": [
```

Comment:
I suggest doing this in code so that we keep the size of our mocks managable. 
Maybe have 2-3 oppgave mocks and just create an array with 200 of each oppgave :)

Reply:
I prefer real mocks because it can reveal more issues locally. For example, some oppgave have long enhet names which caused formatting issues.
Is there any performance problem when we use 155 oppgave data?